CMakeLists.txt

```diff
@@ -1157,16 +1157,16 @@ endif (UNIX)
 #   Release => -O3 -NDEBUG
```

Comment:
Re the PR title `i#4400: Support CFLAGS`

The issue number needs to be corrected.

Reply:
Ooops. But how? Derek asked me to not force push, so I can't fix up an existing commit...

Reply:
Editing the title at the top of the PR page would be sufficient. No need to fix it in the commit.

Reply:
Done